Sources/SwiftSyntaxBuilder/gyb_generated/DeclBuildables.swift

```diff
@@ -0,0 +1,1082 @@
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    deinitKeyword: TokenSyntax = Tokens.`deinit`,
+    body: CodeBlock
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      deinitKeyword: deinitKeyword,
+      body: body
+    )
+  }
+}
+
+extension SubscriptDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    subscriptKeyword: TokenSyntax = Tokens.`subscript`,
+    genericParameterClause: GenericParameterClause? = nil,
+    indices: ParameterClause,
+    result: ReturnClause,
+    genericWhereClause: GenericWhereClause? = nil,
+    accessor: SyntaxBuildable? = nil
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      subscriptKeyword: subscriptKeyword,
+      genericParameterClause: genericParameterClause,
+      indices: indices,
+      result: result,
+      genericWhereClause: genericWhereClause,
+      accessor: accessor
+    )
+  }
+}
+
+extension ImportDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    importTok: TokenSyntax = Tokens.`import`,
+    importKind: TokenSyntax? = nil,
+    @AccessPathBuilder pathBuilder: () -> AccessPath = { .empty }
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifiers: modifiersBuilder(),
+      importTok: importTok,
+      importKind: importKind,
+      path: pathBuilder()
+    )
+  }
+}
+
+extension AccessorDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    modifier: DeclModifier? = nil,
+    accessorKind: TokenSyntax,
+    parameter: AccessorParameter? = nil,
+    asyncKeyword: String?,
+    throwsKeyword: TokenSyntax? = nil,
+    body: CodeBlock? = nil
+  ) {
+    self.init(
+      attributes: attributesBuilder(),
+      modifier: modifier,
+      accessorKind: accessorKind,
+      parameter: parameter,
+      asyncKeyword: SyntaxFactory.makeIdentifier(asyncKeyword),
+      throwsKeyword: throwsKeyword,
+      body: body
+    )
+  }
+}
+
+extension AccessorBlock {
+  public init(
+    leftBrace: TokenSyntax = Tokens.`leftBrace`,
+    @AccessorListBuilder accessorsBuilder: () -> AccessorList = { .empty },
+    rightBrace: TokenSyntax = Tokens.`rightBrace`
+  ) {
+    self.init(
+      leftBrace: leftBrace,
+      accessors: accessorsBuilder(),
+      rightBrace: rightBrace
+    )
+  }
+}
+
+extension VariableDecl {
+  public init(
+    @AttributeListBuilder attributesBuilder: () -> AttributeList = { .empty },
+    @ModifierListBuilder modifiersBuilder: () -> ModifierList = { .empty },
+    letOrVarKeyword: TokenSyntax,
```

Comment:
Again, we can't make a default token here.

Reply:
That’s fine with me.